src/commands/deployWorkspaceProject/internal/startingConfiguration/getStartingConfiguration.ts

```diff
@@ -41,3 +42,16 @@ export async function getStartingConfiguration(context: DeployWorkspaceProjectIn
         environmentVariables: await EnvironmentVariablesListStep.workspaceHasEnvFile() ? undefined : [], // Todo: revisit this
     };
 }
+
+async function tryAddMissingAzureResourcesToContext(context: DeployWorkspaceProjectInternalContext): Promise<void> {
+    if (!context.containerApp && !context.managedEnvironment) {
+        return;
+    } else if (context.containerApp) {
+        const resources = await getResourcesFromContainerAppHelper(context, context.containerApp);
```

Comment:
Can we get the ACR from the container as well? I figured the image name would have it.

Reply:
Yeah, potentially.  I think that logic is complicated enough plus we might have to investigate if that's a good UX so I'll leave that investigation for a different PR/discussion